api/envoy/config/rbac/v2alpha/rbac.proto

```diff
@@ -126,6 +126,27 @@ message Permission {
     // match, this permission would not match. Conversely, if the value of `not_rule` would not
     // match, this permission would match.
     Permission not_rule = 8;
+
+    // The requested TLS SNI server name from the client's connection request.
```

Comment:
Per below this is not necessarily TLS SNI. Should it be something like:

"The request server from the client's connection request. This is typically TLS SNI." (Or something like that?)